src/view/com/notifications/Feed.tsx

```diff
@@ -141,16 +159,26 @@ export function Feed({
             <NotificationFeedLoadingPlaceholder />
           </View>
         )
+      } else if (!tab.types || tab.types.includes(item.type)) {
```

Comment:
f174623ea42b974b3ad9e2707f3ad9371c8d150d changed my initial implementation that used `const filteredItems = React.useMemo(...)` to not rendering the filtered out types.

I figured since `<Feed item={items}>` already holds this list in memory, duplicating it to `filteredItems` would be wasting memory.

(I dont know if `return null` below negatively affects window virtualization, since approximate size is _even more variable_ now).

🏎️ There could be performance optimization by instead rendering the `FeedItem` and perhaps leveraging [content-visibility](https://developer.mozilla.org/en-US/docs/Web/CSS/content-visibility).